src/v2/controllers/FormController.js

```diff
@@ -276,17 +277,16 @@ export default Controller.extend({
       Util.logConsoleError(error);
       errorObj = { responseJSON: { errorSummary: loc('error.unsupported.response', 'login')}};
     }
-    const showErrorBanner = !form?.isFormErrorCalloutCustomized(errorObj);
-    // show error before updating app state.
-    model.trigger('error', model, errorObj, showErrorBanner);
-    idxStateError = Object.assign({}, idxStateError, {hasFormError: true});
 
-    // showErrorBanner : false, suppress the default formError callout, instead let calling View show
-    // custom form error callout
-    if (!showErrorBanner) {
+    if(_.isFunction(form?.showCustomFormErrorCallout)) {
       form.showCustomFormErrorCallout(errorObj);
+      this.showErrorBanner = !!form.showErrorBanner;
     }
 
+    // show error before updating app state.
+    model.trigger('error', model, errorObj, this.showErrorBanner);
+    idxStateError = Object.assign({}, idxStateError, {hasFormError: true});
```

Comment:
Would this work without using instance level properties?

```
let showErrorBanner = false;

if(_.isFunction(form?.showCustomFormErrorCallout)) {
        form.showCustomFormErrorCallout(errorObj);
        showErrorBanner = true;
}

model.trigger('error', model, errorObj, showErrorBanner);
```

The other way we could think of is set `showCustomFormErrorCallout = null`  by default, then

```
let showErrorBanner = _.isFunction(form?.showCustomFormErrorCallout);

if(showErrorBanner) {
        form.showCustomFormErrorCallout(errorObj);
}
```

Reply:
Done